RestAPI.cs

```diff
@@ -43,12 +43,12 @@ public class RestAPI
             if (string.IsNullOrEmpty(url))
                 throw new Exception("Please use a valid WooCommerce Restful API url.");
 
-            url = url.Trim().ToLower().TrimEnd('/');

```

Comment:
Can you check your wordpress setting and see if you got ABC or abc?

![image](https://cloud.githubusercontent.com/assets/6334728/26563370/618e45ae-4525-11e7-836d-ae17cf38452d.png)